apps/docs/app/ui/components/migrateGuide/migrate-guide.css

```diff
@@ -0,0 +1,9 @@
+.hd-migrate-guide code {
```

Comment:
Parsing the mdx and add hd-code to every code elements seemed overly complicated, I decided to duplicate the CSS declaration, if a it's needed a third time a better approach could be discussed.